dotCMS/src/integration-test/java/com/dotcms/datagen/FieldDataGen.java

```diff
@@ -8,16 +8,14 @@
 
 public class FieldDataGen extends AbstractDataGen<Field> {
 
-    private final long currentTime = System.currentTimeMillis();
-
-    private String name = "testFieldName" + currentTime;
+    private String name;
```

Comment:
Codacy found an issue: [Field name has the same name as a method](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6326102)